src/util/std_expr.h

```diff
@@ -44,16 +38,11 @@ class transt:public exprt
   const exprt &trans() const { return op2(); }
 };
 
-/*! \brief Cast a generic exprt to a \ref transt
- *
- * This is an unchecked conversion. \a expr must be known to be \ref
- * transt.
- *
- * \param expr Source expression
- * \return Object of type \ref transt
- *
-
-*/
+/// Cast a generic exprt to a \ref transt
+/// This is an unchecked conversion. \a expr must be known to be \ref
+/// transt.
+/// \param expr Source expression
```

Comment:
@owen-jones-diffblue, comparing with your PR, should we have `:` after the param name?

Reply:
Yes, there should a `:` after the parameter name. This is the format given in `CODING_STANDARD.md`: `\param arg_name: This parameter doesn't need much description`.

Reply:
OK, added everywhere.